doc/introduction/interfaces/numpy.rst

```diff
@@ -174,74 +226,56 @@ How does automatic differentiation work in the case where the QNode returns mult
+        return qml.expval(qml.PauliY(0)), qml.expval(qml.PauliZ(1))
 
 If we were to naively try computing the gradient of ``circuit5`` using the :func:`~.grad` function,
 
-.. code::
-
-    g1 = qml.grad(circuit5, argnum=0)
-    g1(np.pi/2)
+>>> g1 = qml.grad(circuit5)
+>>> params = np.array([np.pi/2, 0.2])
+>>> g1(params)
 
 we would get an error message. This is because the `gradient <https://en.wikipedia.org/wiki/Gradient>`_ is
 only defined for scalar functions, i.e., functions which return a single value. In the case where the QNode
 returns multiple expectation values, the correct differential operator to use is
 the `Jacobian matrix <https://en.wikipedia.org/wiki/Jacobian_matrix_and_determinant>`_.
 This can be accessed in PennyLane as :func:`~.jacobian`:
 
->>> j1 = qml.jacobian(circuit5, argnum=0)
->>> j1(np.pi/2)
-array([-1. -1.])
+>>> j1 = qml.jacobian(circuit5)
+>>> j1(params)
+array([[ 0.        , -0.98006658],
+       [-0.98006658,  0.        ]])
 
 
 The output of :func:`~.jacobian` is a two-dimensional vector, with the first/second element being
 the partial derivative of the first/second expectation value with respect to the input parameter.
-The Jacobian function has the same signature as the gradient function, requiring the user to specify
-which argument should be differentiated.
-
-If you want to compute the Jacobian matrix for a function with multiple input parameters
-and multiple expectation values, the recommended way to do this is to combine the parameters
-into a single list/array and index into this inside your quantum circuit function.
-Consider the following circuit:
-
-.. code-block:: python
-
-    @qml.qnode(dev)
-    def circuit6(params):
-        qml.RX(params[0], wires=0)
-        qml.RZ(params[1], wires=0)
-        qml.RX(params[2], wires=1)
-        qml.CNOT(wires=[0, 1])
-        return qml.expval(qml.PauliZ(0)), qml.expval(qml.PauliZ(1))
-
-It has a full Jacobian with two rows and three columns:
-
->>> j2 = qml.jacobian(circuit6, argnum=0)
->>> params = np.array([np.pi / 3, 0.25, np.pi / 2])
->>> j2(params)
-array([[-8.66025404e-01, -5.55111512e-17,  0.00000000e+00],
-       [-4.71844785e-16, -1.38777878e-17, -5.00000000e-01]])
-
-.. warning::
-
-    Currently, :func:`pennylane.jacobian` supports only the case where
-    ``argnum`` is a single integer. For quantum functions with multiple arguments,
-    use the above method of combining arguments in an array to get the full Jacobian matrix.
 
 
 Advanced Autograd usage
 -----------------------
 
-The PennyLane NumPy interface leverages the Python library `autograd <https://github.com/HIPS/autograd>`_ to enable automatic differentiation of NumPy code, and extends it to provide gradients of quantum circuit functions encapsulated in QNodes. In order to make NumPy code differentiable, Autograd provides a wrapped version of NumPy (exposed in PennyLane as :code:`pennylane.numpy`).
+The PennyLane NumPy interface leverages the Python library `autograd
```

Comment:
This section used the term 'user', and it shouldn't, so I gave it a quick re-write